utilities/AFQ_mrtrixInit.m

```diff
@@ -1,4 +1,4 @@
-function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder)
+function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder, mrtrixVersion)
```

Comment:
I think it makes the most sense for you to fix these changes in your PR.
Then I will check them again. If I start editing code at the same time we
are likely to end up with conflicting versions :-)

Jason D. Yeatman, PhD
Assistant Professor, Institute for Learning & Brain Sciences (I-LABS)
Department of Speech & Hearing Sciences
University of Washington
http://brainandeducation.com

On Thu, Jul 7, 2016 at 2:25 PM, Garikoitz Lerma-Usabiaga <
notifications@github.com> wrote:

> In utilities/AFQ_mrtrixInit.m
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r69990118>:
>
> > @@ -1,4 +1,4 @@
> > -function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder)
> > +function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder, mrtrixVersion)
>
> OK, I will check it in the server instead of my local osx machine. Are you
> going to make changes to the other files or should I commit everything back?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r69990118>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABsYynqxc9DMa2d9u11BQiOc2lIAALa2ks5qTW6sgaJpZM4JGbEU>
> .
>